firestore/src/android/listener_registration_android.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2020 Google LLC
+
 #ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_ANDROID_LISTENER_REGISTRATION_ANDROID_H_
```

Comment:
⚠️ Lint warning: `Lines should be <= 80 characters long`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/src/android/listener_registration_android.h:3:  Lines should be <= 80 characters long  [whitespace/line_length] [2]"></hidden>